data/lib/src/network/model/response/shared_space_node_nested_response.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:data/src/network/model/converter/datetime_converter.dart';
+import 'package:data/src/util/attribute.dart';
+import 'package:domain/domain.dart';
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:data/src/network/model/converter/data_from_json_converter.dart';
+
+part 'shared_space_node_nested_response.g.dart';
+
+@JsonSerializable()
+@DatetimeConverter()
+class SharedSpaceNodeNestedResponse extends Equatable {
+  SharedSpaceNodeNestedResponse(
+    this.sharedSpaceId,
+    this.sharedSpaceRole,
+    this.creationDate,
+    this.modificationDate,
+    this.name,
+    this.nodeType
+  );
+
+  @JsonKey(name: Attribute.uuid, fromJson: sharedSpaceIdFromJson, toJson: sharedSpaceIdToJson)
+  final SharedSpaceId sharedSpaceId;
```

Comment:
I think the data layer should use it's own model. Because in a case the API is changed format response, so we have to refactor all layer include data, domain, presentation layers if we use the same data model for all layers. 
So if we seperate data model between data layers and other layers, when the API is changed, only data layer has to update. 

You can refer to `adapter` pattern for more information.

Reply:
The other thing I want to suggest, I think all data type of attribute of response class should be as same as possible with response json structure. For example:
`SharedSpaceId` should be `String`. Because the response just return:
```
  {
    "uuid": "e352ed55-abef-4630-816f-c025caa3b025",
     ...
  }
```
So when we convert json to response class, we don't have to waste memory to create a new instance of class `SharedSpaceId `.

Reply:
Actually I use SharedSpaceNodeNestedResponse for the API Request and only in Data Layer, I'll convert it to SharedSpaceNodeNested for Domain and Presentation so we separate both models